src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -105,4 +105,103 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function update_additional_data( $attendee, $ticket, $attendee_data ) {
+
+		$attendee_data = $this->format_attendee_data( $attendee, $ticket, $attendee_data );
+
+		try {
+			$query = $this->by_primary_key( 'id', $attendee->ID )
+			              ->set_args( $attendee_data );
+		}
+		catch ( Tribe__Repository__Usage_Error $e ) {
+			do_action( 'tribe_log', 'error', __CLASS__, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: e5fef70ff46924fbd5e42a5406687665</sub>